tests/lib/databases.py

```diff
@@ -63,6 +63,7 @@ def host_url(self):
     def engine(self, dialect=None, **kwargs):
         url = self._url(self.host_from_host, self.port_from_host, include_driver=True)
         engine_url = sqlalchemy.engine.make_url(url)
+        kwargs["echo"] = bool(os.getenv("LOG_SQL"))
```

Comment:
Enabling `echo` here will only effect the test setup queries, not the queries executed by the Query Engine which uses it's own connection object. You're better off relying on the behaviour of `init_logging()` and just making sure you call that whenever `LOG_SQL` is set.

Reply:
Another thing I misunderstood!  I thought what we wanted was to be able to log the sql queries without all the other logging. 
That's done now.

Reply:
Ah, really sorry about this Becky! I think this was one of those "quickly note something down to record the bug" issues which then got turned in to a ticket in Next without making sure it would make sense to someone who didn't have all the original context.

I would say it's all my fault but actually I think it's all Peter's :grinning: 

Let's try and make sure we don't do this again.

Reply:
It's actually fine, I've learned lots along the way about what pytest does with logging :)